plugins/backend-trace-forwarder/src/Cardano/BM/Backend/TraceForwarder.lhs

```diff
@@ -300,24 +300,6 @@ sendItems config tfMVar items@(lo:_) getStateDigest =
     let bs' = BL.toStrict $ encode a
     in (BS.length bs', bs')
 
-  -- Handle is bad, it looks like the connection is broken.
-  -- Check if the queue is almost full.
-  reConnectIfQueueIsAlmostFull = do
```

Comment:
I removed this function for two reasons:

1. Simplify the code.
2. In the real world cases the number of `LogObject`s from the node is very big, so the queue will be "almost full" really fast. I don't see why do we need this function.

Reply:
The original strategy was developed with consideration and was a robust piece -- in that it abstracted over the reasons for why the queue could get stuck and allowed pre-emptive reconnection.

This change eliminates that piece of robustness.

At the same time, IIUC, the problem this change attempts to solve wasn't directly caused by this reconnection approach either -- it's due to `closeHandle` blocking.  So it has to have a separate solution.